src/app/portlets/content-types/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.html

```diff
@@ -34,11 +34,20 @@
         {{ currentFieldType?.label }} <dot-icon-button icon="help_outline" (click)="fieldHelperOverlay.toggle($event)"></dot-icon-button>
     </p-header>
 
-    <dot-content-type-fields-properties-form
-        [formFieldData]="formData"
-        (saveField)="saveFieldsHandler($event)"
-        #fieldPropertiesForm>
-    </dot-content-type-fields-properties-form>
+    <p-tabView [activeIndex]="dialogActiveTab">
+        <p-tabPanel header="{{ i18nMessages['contenttypes.dropzone.tab.overview'] }}">
```

Comment:
I'm not sure if there is any performance issues here, but why

```header="{{ i18nMessages['contenttypes.dropzone.tab.overview'] }}```

instead:

```[header]="i18nMessages['contenttypes.dropzone.tab.overview']```

?